aptos-move/framework/aptos-framework/sources/reconfiguration.move

```diff
@@ -106,14 +105,9 @@ module aptos_framework::reconfiguration {
         borrow_global<Configuration>(@aptos_framework).last_reconfiguration_time
     }
 
-    /// Private function to do reconfiguration.  Updates reconfiguration status resource
+    /// Private function to do reconfiguration. Updates reconfiguration status resource
     /// `Configuration` and emits a `NewEpochEvent`
     fun reconfigure_() acquires Configuration {
-        // Do not do anything if genesis has not finished.
-        if (timestamp::is_genesis() || timestamp::now_microseconds() == 0 || !reconfiguration_enabled()) {
```

Comment:
Sure this can be removed? I remember we needed to add this in Diem.

Reply:
I might have missed something but this is only needed because of consensus_config::set() which also calls reconfigure_(). I removed that call as there's another consensus_config::initialize() call right before that can be passed the configs and will not call reconfigure_().

Reply:
we needed this to avoid multiple reconfig events in genesis

Reply:
I don't think there are multiple reconfig events based on the golden files. Do you have suggestions for double checking?

Reply:
It's there to avoid someone added it, I don't know what'd happen if there's multiple, I hope there's check on rust side

Reply:
Alright I added it back :) Better safe than sorry I guess